### ISSUE TITLE
MDCardSwipe deletes item when clicking on label

### ISSUE THREAD

**Issue by Kalaschnimoe** [kivymd/KivyMD#902]
### Description of the Bug

When using the example of the docs for MDCardSwipe, the bottom layer is clickable/enabled although it is invisible. Therefore the item gets deleted, when the label is clicked on the left side (in the example click on "one" of "one-line item")

Additionally you have to swipe from the very left side to get to the bottom layer. That is impossible on some android phones as swiping from the left has the same functionality as the back key. For example on my oneplus 8t.

### Code and Logs

```python
from kivy.lang import Builder
from kivy.properties import StringProperty

from kivymd.app import MDApp
from kivymd.uix.card import MDCardSwipe

KV = '''
<SwipeToDeleteItem>:
    size_hint_y: None
    height: content.height

    MDCardSwipeLayerBox:
        padding: "8dp"

        MDIconButton:
            icon: "trash-can"
            pos_hint: {"center_y": .5}
            on_release: app.remove_item(root)

    MDCardSwipeFrontBox:

        OneLineListItem:
            id: content
            text: root.text
            _no_ripple_effect: True


Screen:

    BoxLayout:
        orientation: "vertical"
        
        BoxLayout:
            orientation: "vertical"
            spacing: "10dp"
    
            MDToolbar:
                elevation: 10
                title: "MDCardSwipe"
    
            ScrollView:
    
                MDList:
                    id: md_list
                    padding: 0
'''


class SwipeToDeleteItem(MDCardSwipe):
    text = StringProperty()


class TestCard(MDApp):
    def __init__(self, **kwargs):
        super().__init__(**kwargs)
        self.screen = Builder.load_string(KV)

    def build(self):
        return self.screen

    def remove_item(self, instance):
        self.screen.ids.md_list.remove_widget(instance)

    def on_start(self):
        for i in range(20):
            self.screen.ids.md_list.add_widget(
                SwipeToDeleteItem(text=f"One-line item {i}")
            )


TestCard().run()
```

### Screenshots

https://user-images.githubusercontent.com/49235968/110155469-f2d59b00-7de5-11eb-9762-dd434f79363f.mp4

### Versions

* OS: windows
* Python: 3.7
* Kivy: 2.0.0
* KivyMD: master

**Comment by maleksware** [comment]
Did you try to test this on other phones/emulator?

**Comment by Kalaschnimoe** [comment]
Of course. Tested on Windows, Android and emulator

**Comment by thvis** [comment]
I can confirm the issue. I faced the exact same. 

MDIconButtons inside MDCardSwipeLayerBox are enabled even when MDCardSwipeFrontBox is closed

OS: Ubuntu 18
Python: 3.7
Kivy:2.0.0
Kivymd: 0.104.2

@Kalaschnimoe 
You can do a temporary fix until the bug is removed by disabling the buttons with a condition in kv-file. I used the "open_progress" attribute. From Kivymd Doc:  **open_progress** "Percent of visible part of side panel. The percent is specified as a floating point number in the range 0-1. 0.0 if panel is closed and 1.0 if panel is opened." You need to adjust it to your liking. In my case 50% opening of swipe card was good. The other issue, that you need to swipe starting from the very edge did not occur for me when I used the swipe_type: "auto". 

`    MDCardSwipeLayerBox:
        padding: "8dp"

        MDIconButton:
            icon: "trash-can"
            pos_hint: {"center_y": .5}
            on_release: app.remove_item(root)
            disabled: True if root.open_progress < 0.5 else False

    MDCardSwipeFrontBox:

        OneLineListItem:
            id: content
            text: root.text
            _no_ripple_effect: True`

**Comment by HeaTTheatR** [comment]
Fix in master version.
